components/brave_wallet_ui/common/reducers/wallet_reducer.ts

```diff
@@ -387,4 +402,20 @@ reducer.on(WalletActions.setSitePermissions, (state: any, payload: SitePermissio
   }
 })
 
+reducer.on(WalletActions.queueNextTransaction, (state: any) => {
+  const pendingTransactions = sortTransactionByData(state.pendingTransactions)
```

Comment:
Why do we need to sort the pending transactions by created-time if they're always prepended at the front of the list?